### DISCUSSION TITLE
bar chart

### DISCUSSION THREAD

**Discussion by yusufjon-akhmedov** [patrykandpatrick/vico#800] [Questions]
### Question

how to disable vico animation for bar chart I tried everything that I know, could you please help me with that

### UI framework(s)

Jetpack Compose

**Comment by simonalveteg** [comment]
have you tried setting diffAnimationSpec  = snap() as per #341 ? assuming you're using vico 2.

**Reply by yusufjon-akhmedov** [reply]
I did, it didn't work?

**Reply by patrickmichalik** [reply]
@Yusufjon1224, just to confirm, are you referring to the column height animation? If so, the solution mentioned by @simonalveteg should do the trick. If it doesn’t work on your end, could you please share your code and Vico version?

Two notes:

- Since Vico 1.11.0, you can use `null` instead of `snap` to disable animations. This is slightly more memory-efficient.
- Since Vico 2.0.0 Alpha 23, the `diffAnimationSpec` parameter is called `animationSpec`.

Thanks for your input, @simonalveteg!
